src/main/java/Ui.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Ui {
+
+    private Scanner sc;
+
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    public String getNextLine() {
+        return sc.nextLine();
+    }
+
+    public String getNext() {
+        return sc.next();
+    }
+
+    public void printHello() {
+        System.out.println("    _____________________________________");
```

Comment:
Maybe could assign "    _____________________________________" to a String variable and the same variable when needed.

Reply:
Hi thank you for your constructive comment. I did not think of doing that and i think it will really clean up my codes.